src/scripts/main.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.addEventListener('click', e => {
+  const workSpaceY = wall.clientHeight - spider.clientHeight;
+  const workSpaceX = wall.clientWidth - spider.clientWidth;
+
+  let normalizedY = e.offsetY;
+  let normalizedX = e.offsetX;
```

Comment:
i clicked here( spider should move directly to click spot
try to subtract half of the spider here
<img width="277" alt="image" src="https://user-images.githubusercontent.com/66749919/219847169-63e5ed19-9e58-43fe-ab54-2e3bd800e4cb.png">
if you have troubles - fell free to ask in fe_chat)